src/less/charts.less

```diff
@@ -98,13 +98,19 @@
 //Responsive styles for mobiles
 
 @media (max-width: @global-screen-xsmall-max) {
-  .chart-doughnut__info {
-    div {
-      margin-top: @global-whitespace-small !important;
-    }
+  .chart {
+    .chart__header {
+      display: block !important;
 
-    .list-title {
-      font-size: @global-font-size-xsmall !important;
+      .chart-doughnut__info {
+        div {
+          margin-top: @global-whitespace-small !important;
```

Comment:
I don't think this !important accomplishes anything. Let me know if I'm wrong here but removing them has no effect.